.agents/skills/address-pr-comments/SKILL.md

```
@@ -0,0 +1,217 @@
+- [list of changes made]
+"
+```
+
+---
+
+## Step 6: Report Results
+
+Provide a summary to the user:
+
+### Addressed Comments
+| File | Line | Change Made |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)